src/vs/base/common/htmlContent.ts

```diff
@@ -34,6 +34,18 @@ export class MarkdownString implements IMarkdownString {
 	public supportThemeIcons?: boolean;
 	public supportHtml?: boolean;
 	public baseUri?: URI;
+	public uris?: { [href: string]: UriComponents } | undefined;
+
+	public static lift(dto: IMarkdownString): MarkdownString {
```

Comment:
FYI @mjbvz, I don't know if there's an existing pattern for this, but I've messed this up in the past when I want a real MarkdownString for some reason.